src/simulator/simulator.h

```diff
@@ -246,6 +246,40 @@ struct SimulatorMaker
 };
 
 
+template<std::size_t dimension, std::size_t interp_order, std::size_t nbRefinedPart>
+class RuntimerSimulatorCaster
```

Comment:
Why not just `SimulatorCaster` (as you define the alias in the dw) ? I don't get the "Runtimer"

Reply:
sure